packages/adapter-vue/test/unit/lazy-autodirty-scope.spec.ts

```
@@ -0,0 +1,226 @@
+import {
+    defineComponent, 
+    h, 
+    nextTick, 
+    reactive,
+} from 'vue';
+import { Container } from 'validup';
+import type { Validator } from 'validup';
+import { useValidup } from '../../src';
+import type { ValidupComposable } from '../../src';
+
+/* eslint-disable vue/one-component-per-file -- test fixtures defined alongside the spec */
```

ESLint

[error] Definition for rule 'vue/one-component-per-file' was not found.

(vue/one-component-per-file)